sympy/sets/fancysets.py

```diff
@@ -770,6 +770,13 @@ def is_finite_set(self):
             return True
         return self.size.is_finite
 
+    @property
+    def is_empty(self):
+        try:
+            return self.size.is_zero
+        except ValueError:
+            return None
```

Comment:
I don't understand why `self.size` raises `ValueError`. It should be changed to not raise.

Reply:
Well, earlier it raised for any type of symbolic size, now it just raises if the symbols are not integers. (I added a PR some time ago.)

One need to decide what e.g. `Range(a, b, c)` should return. `(a-b)/c`? Which may be negative. So `Max((a-b)/c, 0)`. Which may be fractional. So `Max(floor((a-b)/c), 0)`?